js/main.js

```diff
@@ -63,8 +64,11 @@ function contentLoaded(event) {
   for (let i = 0; i < data.entries.length; i++) {
     const $entry = renderEntry(data.entries[i]);
     $ul.appendChild($entry);
+    viewSwap(data.view);
+    toggleNoEntries();
```

Comment:
These need to be outside your `for` loop, so they aren't being called for every item in the array